src/shared/components/show-hide/ShowHide.scss

```diff
@@ -11,6 +11,6 @@
 }
 
 .chevron {
-  margin-left: 10px;
+  margin-left: var(--show-hide-chevron-margin-left, 10px);
```

Comment:
If you want a variable for configuring this space:

<img src="https://user-images.githubusercontent.com/6834224/182425677-7ce26fe1-ba2d-420c-a30d-dc3fbd25b69f.png" width="150px" />

then it's probably worth calling it something like `show-hide-gutter`. But in any case, it would only make sense if show/hide is rendering with a label. Without a label, just pass a regular class with all the necessary styles to ShowHide.

Reply:
Reverted this change. I don't think we need a separate label for this at the moment.